packages/utilities/fast-web-utilities/src/aria.ts

```diff
@@ -1,4 +1,6 @@
-export enum Orientation {
-    horizontal = "horizontal",
-    vertical = "vertical",
-}
+export const Orientation = {
+    horizontal: "horizontal",
+    vertical: "vertical",
+} as const;
+
+export type Orientation = typeof Orientation[keyof typeof Orientation];
```

Comment:
This change feels like it should be in its own PR, or at least grouped with similar changes.  It's impact goes beyond our storybook. 
Does this risk breaking folks who have a dependency on the 'enum' type?
Wouldn't it be simpler to define the exported type like so?
`export type Orientation = "vertical" | "horizontal";`

Reply:
There should be no major differences between using an enum vs an object: https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums

Reply:
> Wouldn't it be simpler to define the exported type like so?

Explicit types can't be used as values, (ex. `Object.values(Orientation)`). It's useful when setting arg array values for Storybook controls.